src/app/modules/declare-expenses/service/declare-expenses.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Account } from 'src/app/models/account';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DeclareExpensesService {
+
+  url:string = "localhost:4200/personalfinance";
```

Comment:
Okay for now, but you should leverage the `environment.ts` instead of hard-coding this value.